### DISCUSSION TITLE
Child processes won't elevate

### DISCUSSION THREAD

**Discussion by DailenG** [Hofknecht/SystemTrayMenu#487] [Q&A]
I'm running System Tray Menu on a user with no admin privileges and using shortcuts that launch chocolatey to install an application. I have SystemTrayMenu running with elevated privileges, I have chocolatey set to always run with admin privileges, but if I launch a shortcut from System Tray Menu it launches without elevation or prompting for elevation (either is ok). If I launch the shortcut directly it will prompt for elevation.

Am I doing something wrong or is there a workaround? I'm not sure how to get it to prompt aside from using a third party like gsudo to induce elevation.

**Comment by skorpioninka** [comment]
Hello, I have the same problem.
Have you found a solution?
